src/main/java/org/springframework/samples/petclinic/rest/importcsv/ImportCSV.java

```diff
@@ -32,27 +32,18 @@
         consumes = "text/plain",
         produces = "application/json")
     public ResponseEntity<List<Pet>> importPets(@RequestBody String csv) {
```

Comment:
when you refactor a long method, try to extract (small) methods for each intention right at the beginning of your refactoring session.
you can also have a look here, which is basically a copy of the contents of Martin Fowler's Refactoring book: [https://refactoring.guru/extract-method](https://refactoring.guru/extract-method)